tests/output/webpage/assert_test.js

```diff
@@ -170,20 +175,23 @@ describe("scriptTest tests", function() {
         title: "scriptTest reports success for matching code",
         code: "<div><script>var x = 4;</script></div>",
         validate: xTest,
-    }); 
+    });
 
     assertTest({
         title: "scriptTest reports failure for not matching code",
         code: "<div></div>",
```

Comment:
I wish the titles were more descriptive.  I assume that we don't like empty tags or empty tags that aren't self-closing.

Reply:
Ah, this is a test result failure, not a lint result failure. Yeah, I did totally reword a bunch of test titles in my big PR, as I prefer verbosity in test descriptions. (There are a bunch of one-word test titles, uck). There might be a PR for that upcoming. :)

Reply:
And by test, I mean validation test, challenge test. The word "test" gets confusing here.